start.js

```diff
@@ -79,37 +78,10 @@ function stop (error) {
   process.exit()
 }
 
-function parsedArgs (args) {
-  return argv(args, {
-    number: ['port', 'body-limit'],
-    boolean: ['pretty-logs', 'options', 'watch'],
-    string: ['log-level', 'address'],
-    envPrefix: 'FASTIFY_',
-    alias: {
-      port: ['p'],
-      socket: ['s'],
-      help: ['h'],
-      options: ['o'],
-      address: ['a'],
-      watch: ['w'],
-      prefix: ['r'],
-      'log-level': ['l'],
-      'pretty-logs': ['P'],
-      'plugin-timeout': ['T']
-    },
-    default: {
-      'pretty-logs': false,
-      'watch': false,
-      'options': false
-    }
-  })
-}
-
 function runFastify (args, cb) {
   require('dotenv').config()
-  let opts = parsedArgs(args)
+  let opts = parseArgs(args)
   opts.port = opts.port || process.env.PORT || 3000
```

Comment:
@mcollina should this be `PORT` or `FASTIFY_PORT` ? I think the latter right?

Reply:
it should be PORT. it’s a well known env variable

Reply:
Ok, I removed it from README initially so adding it back now